reportportal_client/helpers.py

```diff
@@ -11,14 +11,47 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
-
 import logging
-from pkg_resources import DistributionNotFound, get_distribution
+import uuid
 from platform import machine, processor, system
 
+from pkg_resources import DistributionNotFound, get_distribution
+
+import six
+
 logger = logging.getLogger(__name__)
 
 
+def generate_uuid():
+    """Generate Uuid."""
+    return str(uuid.uuid4())
+
+
+def convert_string(value):
+    """Support and convert strings in py2 and py3.
+
+    :param value: input string
+    :return value: convert string
+    """
+    if isinstance(value, six.text_type):
+        # Don't try to encode 'unicode' in Python 2.
+        return value
+    return str(value)
+
+
+def dict_to_payload(dictionary):
+    """Convert dict to list of dicts.
+
+    :param dictionary: initial dict
```

Comment:
Align parameter descriptions, please.

Reply:
I am sorry, but I 'm not sure what exactly do you mean here. Could you, please, clarify your point?

Reply:
```
def dict_to_payload(dictionary):
    """Convert dict to list of dicts.

    :param dictionary: initial dict
    :return list:             list of dicts
    """
```